app/javascript/packs/loader.js

```diff
@@ -0,0 +1,378 @@
+;(function (global) {
+  'use strict'
+
+  var $ = global.jQuery
```

Comment:
I think this could import from jQuery, rather then relying on globals

Reply:
Fixed